tests/MovingImage/Test/Client/VMPro/ApiClient/Guzzle6ApiClientTest.php

```diff
@@ -51,7 +53,17 @@ public function setUp(): void
         $this->serializer = SerializerBuilder::create()->build();
     }
 
-    public function testDoRequest()
+    public function testGetChannelsWithLocaleRequest(): void
+    {
+        $client = new Guzzle6ApiClient($this->httpClient, $this->serializer);
+        $client->getChannels(5, 'en');
+
+        $this->assertCount(1, $this->historyContainer);
+        $this->assertEquals('GET', $this->historyContainer[0]['request']->getMethod());
+        $this->assertEquals('5/channels?locale=en', $this->historyContainer[0]['request']->getUri());
```

Comment:
This tests verifies that the url and it's query args are the same as expected in VMPro
https://github.com/movingimage-evp/mi24-vmpro/pull/684/files#diff-d58f1e5d49eb3fa7f5cc167b74b641f87488d2a91cd5730aaa7d1078298ea6a1R84